lib/gcra/redis_store.rb

```diff
@@ -22,10 +23,12 @@ def initialize(redis, key_prefix)
     # Returns the value of the key or nil, if it isn't in the store.
     # Also returns the time from the Redis server, with microsecond precision.
     def get_with_time(key)
-      time_response = @redis.time # returns tuple (seconds since epoch, microseconds)
+      time_response, value = @redis.pipelined do
```

Comment:
curious, what is the benefit of setting value with `@redis.pipelined` like this? less calls to key store?

Reply:
See https://redis.io/topics/pipelining.  This sends both commands in a single network payload and then decodes both responses.  It reduces the number of network roundtrips, although from the redis server's point of view it is still two commands.  But redis is so fast that the network round trip time always vastly dominates the time taken to execute commands.